<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPregunta</class>
 <widget class="QWidget" name="WidgetPregunta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>137</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QFrame" name="GroupCuello">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">border-color: rgb(108, 108, 108);</string>
        </property>
        <property name="title">
         <string>Cuello</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <property name="horizontalSpacing">
            <number>2</number>
           </property>
           <property name="verticalSpacing">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Izquierdo</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <item>
                <property name="text">
                 <string>Sin dolor</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Leve</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Moderado</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Intenso</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_3">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <item>
                <property name="text">
                 <string>Nunca</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ocasional</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Frecuente</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Continuo</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Derecho</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QComboBox" name="comboBox_4">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <item>
                <property name="text">
                 <string>Sin dolor</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Leve</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Moderado</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Intenso</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_5">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <item>
                <property name="text">
                 <string>Nunca</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ocasional</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Frecuente</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Continuo</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QSlider" name="horizontalSlider">
             <property name="maximum">
              <number>4</number>
             </property>
             <property name="sliderPosition">
              <number>2</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
